rosidl_generator_py/package.xml

```diff
@@ -36,6 +36,14 @@
   <test_depend>python_cmake_module</test_depend>
   <test_depend>rmw</test_depend>
   <test_depend>rosidl_cmake</test_depend>
+
+  <!--
+  Bloom does not support group_depend so entries below duplicate the group rosidl_typesupport_c_packages.
+  -->
+  <test_depend>rosidl_typesupport_connext_c</test_depend>
+  <test_depend>rosidl_typesupport_introspection_c</test_depend>
+  <test_depend>rosidl_typesupport_fastrtps_c</test_depend>
+  <!-- end of group dependencies added for bloom -->
```

Comment:
Q: should we also add  `<group_depend>rosidl_typesupport_c_packages</group_depend>` ?

Reply:
What for? Above we already `buildtool_export` depend on `rosidl_typesupport_c` which has such a group dependency.

Reply:
Because it seems like it is necessary for this package to build (specifically the tests). It seems more correct than relying on it as a transitive dependency.

Reply:
But a group dependency is not limited to tests but acts as a build as well as run dependency.

Reply:
Yeah, so maybe it's best to leave it at the separate test_depend tags.

Reply:
Should instead https://github.com/ros2/rosidl_typesupport/blob/3c15da21132af47d73a3fea00f2d99352e647f11/rosidl_typesupport_c/package.xml#L19-L20 be modified to include `rosidl_typesupport_fastrtps_c` and this package woulld only `test_depend` on `rosidl_typesupport_c` (as it also does `buildtool_export_depend` on)?

Reply:
Maybe I'm wrong, but I don't think the issue is that `rosidl_typesupport_fastrtps_c` is missing. I think the issue is that no C typesupports are depended on here (I presume the build depends of `rosidl_typesupport_c` don't get installed when building `rosidl_generator_py`).

I guess adding one or more `<build_export_depend>` tags for C typesupport packages in `rosidl_typesupport_c` would fix the build though.

Reply:
> I guess adding one or more `<build_export_depend>` tags for C typesupport packages in `rosidl_typesupport_c` would fix the build though.

That doesn't seem right from the perspective of that package.

How about we keep the patch as it is right now and see if it fixes the build as desired?

Reply:
Sure. Considering that the PR job passes (when past PR jobs failed) I'm fairly confident it will work :crossed_fingers: